pgcli/pgcompleter.py

```diff
@@ -43,6 +49,20 @@ def __init__(self, smart_completion=True, pgspecial=None):
 
         self.all_completions = set(self.keywords + self.functions)
 
+        self.suggestion_matchers = {
```

Comment:
This should be made into a class var, since this is a static mapping of keywords to functions and it won't change between objects.